controllers/loginController/index.js

```diff
@@ -0,0 +1,41 @@
+const passport = require('passport');
+require('../../passportConfig')(passport);
+const getDbConnection = require("../../dbConnection");
+
+function login(req, res, next) {
+    getDbConnection();
+    passport.authenticate('local', (err,user,info) => {
+        if(err) {
+            return res.status(500).json({
+                success: false,
+                message: 'Ooops something happened',
+                info: ''
+            })
+        }
+        if(!user) {
+            return res.json({
+                success: false,
+                message: "No User exists",
+                info: ''
+            });
+        }
+
+        //persistent login
```

Comment:
You might want to write a longer comment - I don't know how "persistent login" relates to the code below